xarray/__init__.py

```diff
@@ -104,6 +105,7 @@
     "IndexSelResult",
     "IndexVariable",
     "Variable",
+    "NamedArray",
```

Comment:
are we ready?

Reply:
from offline conversatios: we've decided to keep this and refrain from advertising it to the public until it is fully ready